operator/controllers/http/app.go

```diff
@@ -41,6 +41,17 @@ func (r *HTTPScaledObjectReconciler) createOrUpdateApplicationResources(
 			"Identified HTTPScaledObject creation signal"),
 	)
 
+	// We want to integrate http scaler with other
+	// scalers. when SkipScaledObjectCreation is set to true,
+	// reconciler will skip the KEDA core ScaledObjects creation.
+	// you can create your own so, and add http scaler as one of your triggers.
+	if httpso.Annotations["skipScaledObjectCreation"] == "true" {
+		logger.Info(
+			"Skip scaled objects creation with flag SkipScaledObjectCreation=true",
+			"HTTPScaledObject", httpso.Name)
+		return nil
+	}
+
```

Comment:
@JorTurFer I've a had a look at this,  I have a slight concern in that if you were to create your own ScaledObject with the same name at the same time as adding the annotation on the httpso as part of a helm upgrade etc,  there might be a possibility that the scaledObject could get updated but keep the ownerReferences on the object resulting in the scaledObject getting removed unintentionally by the operator when it checks for the annotation & ownerReferences.  The logic I've tested is scaledObject will get removed if kind=httpscaledobject and and name matches the same name of httpso in ownerreferences.   I'm happy to make these changes but would like to hear your thoughts?

Reply:
you are right in that point, but not removing the old one can generate problems:
- KEDA admission webhooks will reject the new one as the workload is already scaled by the old one (if they use different names)
- If both are at the same time in the cluster, they will scale the same workload, generating unexpected scaling behaviors
- The old one will have the current HTTPScaledObject as owner, so if you remove the HTTP scaler but you maintain the same ScaledObject, removing the HTTPScaledObject will remove the ScaledObject, although they are detached

Honestly, I don't have a good way to do this is in a single step :/

Maybe we cloud introduce an annotation in the ScaledObject that the HTTP add-on operator checks before updating it? 🤔  Maybe we can check both resources (HTTPScaledObject and ScaledObject) and if the HTTPScaledObject sets `skipScaledObjectCreation:true`, we only modify/delete the ScaledObject when the ownership reference is correct.

IDK if this will work as I expect tbh. WDYT?

Reply:
Ok, I already have above solution working which checks the scaledObject  for the ownerReferences and deletes object if kind=httpScaledObject and name matches the httpso name when annotation is set on the httpso.  I was wondering if another (less risky) option would be to instead of deleting the scaledObject, the operator could just remove the OwnerReferences on the scaledObject and the existing scaledObject can then be taken over to self-manage which would ensure that scaledObject does not need to be replaced.  Thoughts?

Reply:
The problem is that we should ensure that resources are removed correctly or we could keep dangling resources :/ 

Maybe we could just remove the reference but in that case, we should document this case well to prevent future issues complaining about that